solr_conf/schema.xml

```diff
@@ -512,51 +512,62 @@
    <field name="xml" 
           type="string" indexed="false" stored="true" multiValued="false"/>
    
+   <!-- default, catch all search field -->
+   <field name="text" 
+          type="text" indexed="true" stored="false" multiValued="true"/>
+   
```

Comment:
@afred: There might be a bit more to it, but one problem with concatenating all your text and putting it in a single field is that you will get adjacent tokens that weren't adjacent in the original document, and you could get false matches when phrase searching. (This is the same on sales and AAPB.)